akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -245,6 +245,22 @@ object ClusterEvent {
   final case class ReachableMember(member: Member) extends ReachabilityEvent
 
   /**
+   * Marker interface to facilitate subscription of
+   * both [[UnreachableDataCenter]] and [[ReachableDataCenter]].
+   */
+  sealed trait DataCenterReachabilityEvent extends ClusterDomainEvent
+
+  /**
+   * A data center is considered as unreachable when any members from the data center are unreachable
+   */
+  final case class UnreachableDataCenter(dataCenter: DataCenter) extends DataCenterReachabilityEvent
```

Comment:
good enough for now, I think those will gain more stats like information in the near future hmm